go/master/c/client.go

```diff
@@ -101,6 +100,12 @@ func paddle_release_master_client(client C.paddle_master_client) {
 	remove(client)
 }
 
+//export paddle_start_get_records
+func paddle_start_get_records(client C.paddle_master_client, pass C.int) {
```

Comment:
How does trainer know which pass is the training currently in? We need someway to get this information, maybe `paddle_set_dataset` could return this information, or creating a new function seems fine as well.